util/linux_install.sh

```diff
@@ -78,7 +78,7 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 	read -r answer
 	if echo "$answer" | grep -iq "^y"; then
 		sudo touch /etc/portage/package.use/qmkfirmware
-		echo "sys-devel/gcc multilib" >> /etc/portage/package.use/qmkfirmware
+		sudo echo "sys-devel/gcc multilib" >> /etc/portage/package.use/qmkfirmware
```

Comment:
The shell redirection here is still invoked with user permissions, so it'll fail.

Reply:
Oh, so the pipe into `tee` is an idiom to get around that? TIL.

I'll revert my changes and add a comment to clarify that. Thanks again!